src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -1181,23 +1181,23 @@ void LedgerConsensusImp<Traits>::propose ()
     JLOG (j_.trace()) << "We propose: " <<
         (ourPosition_->isBowOut ()
             ? std::string ("bowOut")
-            : to_string (ourPosition_->getCurrentHash ()));
+            : to_string (ourPosition_->position ()));
     protocol::TMProposeSet prop;
 
-    prop.set_currenttxhash (ourPosition_->getCurrentHash ().begin ()
+    prop.set_currenttxhash (ourPosition_->position ().begin ()
         , 256 / 8);
```

Comment:
Consider `ourPosition_->position().size()` here (and elsewhere), and avoid the magic number.